Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Microsoft Translator Service/MicrosoftTranslatorPage.xaml

```diff
@@ -0,0 +1,52 @@
+﻿<Page
```

Comment:
This needs running through xaml styler, please see https://github.com/Microsoft/UWPCommunityToolkit/blob/dev/contributing.md#quality-insurance-for-pull-requests-for-xaml-controls